### PR TITLE
added the CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by Rakteem007**
Signed-off-by: Rakteem Mullick <rm1011.coding@gmail.com>

issue link -> <a href="https://github.com/seanohue/vue-howler-bus/issues/7">#7</a>

Added the CONTRIBUTING.md file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  - [Your First Code Contribution](#your-first-code-contribution)
+  - [Improving The Documentation](#improving-the-documentation)
+- [Styleguides](#styleguides)
+  - [Commit Messages](#commit-messages)
+- [Join The Project Team](#join-the-project-team)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Code of Conduct].
```
Comment (seanohue) [inline CONTRIBUTING.md]:
```suggestion
[Code of Conduct](./CODE_OF_CONDUCT.md).
```

This should make it link to the current COC

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+By participating, you are expected to uphold this code. Please report unacceptable behavior
+to <>.
```
Comment (seanohue) [inline CONTRIBUTING.md]:
```suggestion
to [@seanohue](https://github.com/seanohue).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+
+### Your First Code Contribution
+
+Fork the repository. 
+
+```
+
+- git clone <your-forked-repo-link>
+-git checkout -b <appropiate branch name>
```
Comment (seanohue) [inline CONTRIBUTING.md]:
```suggestion
-git checkout -b <appropriate branch name>
```

small typo here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+(for multiple files)
+-git add . or -git add -A
+-git commit -s -m "message"
+-git push -u origin <your-branch-name>
+## Join The Project Team
+<!-- TODO -->
+
+<!-- omit in toc -->
+
+feel free to ask the maintainer in the issue comment section to for the issue and hence joining the team.
```
Comment (seanohue) [inline CONTRIBUTING.md]:
```suggestion
Feel free to ask the maintainer to assign you to an open issue and hence join the team. 
```
Worded a bit clearer this way

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+<!-- TODO -->
+
+<!-- omit in toc -->
+
+feel free to ask the maintainer in the issue comment section to for the issue and hence joining the team. 
+## Attribution
+
+This guide is based on the **contributing-gen**. [Make your own](https://github.com/bttger/contributing-gen)!
+
+Note : The document is adapted from [Torus](https://github.com/torus-online/torus/blob/master/CONTRIBUTING.MD).
```
Comment (seanohue) [inline CONTRIBUTING.md]:
```suggestion
Note: The document is adapted from [Torus](https://github.com/torus-online/torus/blob/master/CONTRIBUTING.MD).
```
No need to have a space before the `:` punctuation

**Comment by Rakteem007** [comment]
okk sir, I will resolve the conflicts @seanohue

**Comment by Rakteem007** [comment]
@seanohue , I have resolved the required changes. Please check the file.

**Review by seanohue — APPROVED** [review]

**Pull request merged**